### ISSUE TITLE
Mach Speed calculated on : air_speed or ground_speed ?

### ISSUE THREAD

**Issue by QuentinG50** [Estia-advanced-programming/pandora-public#28]

**Comment by WilliamDelamare** [comment]
This should be the actual jet's speed (ground speed).

**Comment by Altorel** [comment]
I would like to differ on Mr Delamare's recommendation. Mach speed is based on the flow of air around the aircraft, since it describes how close/far you are from/past the sound barrier. Mach speed usual value of 1243kph is actually valid for standard atmospheric pressure and ambient temperature, as well as a number of other parameters such as humidity...
This is why I think that the airspeed is more indicated to calculate mach speed.

**Comment by dhmmasson** [comment]
I agree with @Altorel comment, see [mach number](https://en.wikipedia.org/wiki/Mach_number)

Mach speed should thus be calculated with air speed.  I'll be updating the test suite accordingly.

**Comment by WilliamDelamare** [comment]
I disagree, but my source is only the wikipedia formula stating to consider the `velocity of the moving aircraft`:
![image](https://user-images.githubusercontent.com/14878561/111997563-73e99d80-8b1b-11eb-8bf1-0f7e74f31fc4.png)

**Comment by dhmmasson** [comment]
In [mach number](https://en.wikipedia.org/wiki/Mach_number) : 
> Pilots of high-altitude aerospace vehicles use flight Mach number to express a vehicle's true **airspeed,**
